README.md

```diff
@@ -28,6 +28,17 @@ $ npm install babel-plugin-syntax-dynamic-import --save-dev
 }
 ```
 
+You can also use `parseOpts` and not install the syntax-plugin:
+
+```json
+{
+  "parserOpts": {
+    "plugins": ["dynamicImport"]
```

Comment:
@hzoo Personally this seems like terrible user experience. I was pro `parserOpts` because you wanted to use it for custom parsers, but I really don't think we should promote it's usage for Babel with Babylon.